.github/workflows/platform.dependencies.yml

```diff
@@ -386,17 +398,31 @@ jobs:
         uses: azure/powershell@v1
         with:
           inlineScript: |
+
+            # Load used functions
+            . (Join-Path $env:GITHUB_WORKSPACE 'utilities' 'pipelines' 'tokensReplacement' 'Convert-TokensInParameterFile.ps1')
+
+            # Prepare Replace Tokens in Parameter File
+            $Settings = Get-Content -Path "settings.json" | ConvertFrom-Json
+            $ConvertTokensInputs = @{
+                LocalCustomParameterFileTokens    = $Settings.parameterFileTokens.localTokens.tokens
+                TokenPrefix                       = $Settings.parameterFileTokens.tokenPrefix
+                TokenSuffix                       = $Settings.parameterFileTokens.tokenSuffix
+            }
+
             # Retrieving parameters from previous job outputs and parameter files
             Write-Verbose "Retrieving parameters from previous job outputs" -Verbose
             $imageTemplateName = '${{ needs.job_deploy_imgt.outputs.imageTemplateName }}'
             $imageTemplateResourceGroup = '${{ needs.job_deploy_imgt.outputs.imageTemplateResourceGroup }}'
 
             Write-Verbose "Retrieving parameters from storage account parameter files" -Verbose
             $parameterFilePath = Join-Path $env:GITHUB_WORKSPACE '${{ env.dependencyPath }}' '${{ env.saNamespace }}' 'parameters' 'parameters.json'
+            $null = Convert-TokensInParameterFile @ConvertTokensInputs -ParameterFilePath $parameterFilePath -Verbose
```

Comment:
Should we run the conversion on all parameter files or twice on each parameter file as done for other tasks?

Reply:
as in creating a task dedicated to process all parameter files in the stage / job? I thought of another option is to construct the parameter files as an array, and then do a foreach. And if we need to do more files, we just add them to the array. To my knowledge, changes do not propagate between jobs, so we need an explicit task per job right?

Reply:
Nevermind I just see this is solved by lines 420 and 425. Resolving the comment